build.gradle.kts

```diff
@@ -0,0 +1,116 @@
+import org.jetbrains.intellij.platform.gradle.IntelliJPlatformType
+import org.jetbrains.intellij.platform.gradle.TestFrameworkType
+import org.jetbrains.intellij.platform.gradle.models.ProductRelease
+
+plugins {
+    alias(libs.plugins.gradleIntelliJPlugin) // Gradle IntelliJ Plugin
+    alias(libs.plugins.kotlinJvm)
+    id("idea")
+}
+
+group = "com.redhat.devtools"
```

Comment:
@sbouchet, @fbricon, @btjd:
wondering if we should/can use a FOSS identifier: the rule of thumb is that @Red Hat we'd use org. for FOSS project while we'd use "com.redhat" for productized projects. This makes a difference ex. when we want to publish to Maven Central where they'd only accept FOSS.
Thoughts?

Reply:
Sure, central is not to be used. It's more of a showcase that it matters. Dont have a concrete example of impediment.